ui/src/containers/SituationList.vue

```diff
@@ -188,7 +196,13 @@ const resetFilters = () => {
 
 <template>
 	<div class="list-main">
-		<h2>Situation List</h2>
+		<div class="header">
+			<h2>Situation List</h2>
+			<FeatherButton class="new-situation-btn" @click="() => addNewSituation()">
```

Comment:
small thing but there are a few areas where @click functions call an extra in-between function.
should be able to just do `@click="addNewSituation"`